sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/GroupingSqlAggregator.java

```diff
@@ -90,7 +90,10 @@ public Aggregation toDruidAggregation(
         }
       }
     }
-    AggregatorFactory factory = new GroupingAggregatorFactory(name, arguments);
+    AggregatorFactory factory = new GroupingAggregatorFactory(
+        name,
+        arguments.stream().distinct().collect(Collectors.toList())
```

Comment:
wouldn't this will mask the newly added check in `GroupingAggregatorFactory` ?

Reply:
made this a soft exception and removed this distinct masking here